### PR TITLE
Post-release version(v1.3.7) update

### PR DESCRIPTION
**Pull request by Ashley-wenyizha**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

**Review by Cappuccinuo — APPROVED** [review]
/lgtm

File: charts/aws-efs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
 version: 2.2.3
```
Comment (wongma7) [inline charts/aws-efs-csi-driver/Chart.yaml:3]: this should be bumped to 2.2.4

Comment (wongma7) [reply charts/aws-efs-csi-driver/Chart.yaml:3]: or I guess 2.2.5. 

since master branch is source of truth wrt helm charts

**Pull request closed without merging**